scripts/ipfs/ipfs-http-gateway.conf

```diff
@@ -0,0 +1,15 @@
+    location /ipfs {
```

Comment:
Previous config was wrapped in a server tag.
Since i need to co-exist/share the same server with multiple module, i created a server tag that includes theses files into it, making one sever tag with multiple modules.

Also changed it from proxying / to individual paths.